### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by sirghiny**
This pull request updates the repository by implementing feedback.

### CODE REVIEW

File: src/SignIn.js [hunk 1]
```diff
@@ -44,17 +45,25 @@ class SignIn extends Component {
       window.sessionStorage.auth = `Bearer ${this.state.token}`;
       this.props.history.push('/home')
     }
-    
-
+    const mainDivStyle = {
```
Comment (Kachulio1) [inline src/SignIn.js]: Maybe we can have our CSS in a separate file

File: src/View_Bucketlist.js [hunk 2]
```diff
@@ -32,14 +50,14 @@ class ViewBucketlist extends Component {
       }.bind(this))
```
Comment (Kachulio1) [inline src/View_Bucketlist.js]: remove the new line

File: src/View_Bucketlist.js [hunk 3]
```diff
@@ -54,58 +72,59 @@ class ViewBucketlist extends Component {
...
+        this.setState({delete_successful: response.data["deleted_bucketlist"]})
+        this.props.history.push('/home')
       }.bind(this))
       .catch(function(error) {
-        this.setState({delete_message: error.response.data["error"]})
+        this.setState({delete_failed: error.response.data["error"]})
       }.bind(this))
   }
-  
+
```
Comment (Kachulio1) [inline src/View_Bucketlist.js]: remove one line blank line here

File: src/View_Bucketlist.js [hunk 4]
```diff
@@ -1,23 +1,41 @@
...
+      delete_failed: "",
     }
   }
+  
+
+  componentWillMount(){
+    this.setState({id: this.props.match.params.id})
+    this.getBucketlist(this.props.match.params.id)
+    this.getItems(this.props.match.params.id)
+    console.log(this.state.id)
```
Comment (Kachulio1) [inline src/View_Bucketlist.js]: remove all the `console.log`

File: src/View_Bucketlist.js [hunk 5]
```diff
@@ -54,58 +72,59 @@ class ViewBucketlist extends Component {
...
   render() {
-    //add api call for bucketlist
-    this.getBucketlist(this.props.location.id)
-    this.getItems(this.props.location.id)
-    var bucketlist = this.state.bucketlist
-    var items = this.state.items
-
+   var bucketlist = this.state.bucketlist
+   var items = this.state.items
+    //var items = this.state.items
```
Comment (Kachulio1) [inline src/View_Bucketlist.js]: remove the commented out code

**Comment by sirghiny** [comment]
I've made all changes except moving css to an external file (I've left them inline). I have been running into an error that causes css selectors for classes not to work (Only selectors for attributes work).

File: src/Edit_Bucketlist.js [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+    }
+  }
+
+  componentWillMount() {
+    this.setState({id: this.props.match.params.id})
+    this.getBucketlist(this.props.match.params.id)
+    this.getItems(this.props.match.params.id)
+  }
+  
+  getBucketlist(bucketlist_id) {
```
[the remark this answers is not in the document]
Comment (sirghiny) [reply src/Edit_Bucketlist.js]: Done, I've changed all to camelCase.

File: src/Edit_Bucketlist.js [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+    function getBucketlistRequest() {
+      var url = 'http://127.0.0.1:5000/api/v1/bucketlists/' + bucketlist_id
```
[the remark this answers is not in the document]
Comment (sirghiny) [reply src/Edit_Bucketlist.js]: Not yet, but I'll add this after researching a bit.

File: src/Edit_Bucketlist.js [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+        url: url,
+        responseType: 'json',
+        headers: {Authorization: window.sessionStorage.auth}
+      })
+    }
+
+    getBucketlistRequest(bucketlist_id)
+      .then(function (response){
+        this.setState({bucketlist: response.data["bucketlist"], error: ""})
+      }.bind(this))
```
[the remark this answers is not in the document]
Comment (sirghiny) [reply src/Edit_Bucketlist.js]: Done.

File: src/Edit_Bucketlist.js [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+
```
[the remark this answers is not in the document]
Comment (sirghiny) [reply src/Edit_Bucketlist.js]: Done

File: src/Edit_Bucketlist.js [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+      .catch(function (error){
+        this.setState({bucketlist: "", getBucketlistError: error.response.data["error"]})
+      }.bind(this))
+
+    }
+
+  getItems(bucketlist_id) {
+    function getItemsRequest(bucketlist_id) {
+      var url = 'http://127.0.0.1:5000/api/v1/bucketlists/' + bucketlist_id + '/items/'
```
[the remark this answers is not in the document]
Comment (sirghiny) [reply src/Edit_Bucketlist.js]: Done.

File: src/Edit_Bucketlist.js [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+        data: data,
+        responseType: 'json',
+        headers: {Authorization: window.sessionStorage.auth}
+      })
+    }
+
+    addItemRequest(this.state.id, item)
+      .then(function (response){
+        var items = this.state.items
+        var new_len = items.push(item)
```
[the remark this answers is not in the document]
Comment (sirghiny) [reply src/Edit_Bucketlist.js]: Done, I've made the urls 'const' and all other variables 'let'

File: src/View_Bucketlist.js (hunk 4 above)
Comment (sirghiny) [reply src/View_Bucketlist.js]: Done.

File: src/View_Bucketlist.js (hunk 2 above)
Comment (sirghiny) [reply src/View_Bucketlist.js]: Done.

File: src/View_Bucketlist.js (hunk 3 above)
Comment (sirghiny) [reply src/View_Bucketlist.js]: Done.

File: src/View_Bucketlist.js (hunk 5 above)
Comment (sirghiny) [reply src/View_Bucketlist.js]: Done.

**Pull request merged**